schema.sql

```diff
@@ -46,3 +46,5 @@ CREATE TABLE visits(
     animal_id INT REFERENCES animals(id),
     visit_date DATE
     );
+    -- Add an email column to your owners table
+ALTER TABLE owners ADD COLUMN email VARCHAR(120);
```

Comment:
Required

- Great work so far on finding this solution to enhance the performance of your app. However, in this case, the commonly used solution is to add indexes to the queries and it's amazingly enhancing the performance. Please try it out, update your screenshots with the new ones, and ask for a re review.
```
   CREATE INDEX ON visits (animal_id);
   CREATE INDEX ON visits (vet_id);
   CREATE INDEX ON owners (email);
```